Readme.adoc

```diff
@@ -1725,7 +1725,9 @@ os.call(cmd: os.Shellable,
         mergeErrIntoOut: Boolean = false,
         timeout: Long = Long.MaxValue,
         check: Boolean = true,
-        propagateEnv: Boolean = true): os.CommandResult
+        propagateEnv: Boolean = true,
+        shutdownGracePeriod: Long = 100,
```

Comment:
What's the time unit used for `shutdownGracePeriod`? Should we name it `shutdownGracePeriodMsec`?

Reply:
I'll probably leave it as is, the scaladoc says it's in milliseconds so that should be enough